test/installation_node/install from source with thread parameter

```diff
@@ -25,8 +25,11 @@ nvm run $NVM_TEST_VERSION --version | grep $NVM_TEST_VERSION || "'nvm run $NVM_T
 # Remove the stuff we're clobbering.
 [ -e ../../$NVM_TEST_VERSION ] && rm -R ../../$NVM_TEST_VERSION
 
-# Install from source with 2 make jobs (and swapped arg order)
-nvm install -j 2 -s $NVM_TEST_VERSION || die "'nvm install -s $NVM_TEST_VERSION' failed"
+# Speed up test for non-CI environment which may have more CPU threads
+[ -z "${CI-}"] && nvm_get_make_jobs
```

Comment:
What I mean is, it's a bad thing when tests do different things in different environments; it means local won't be an accurate predictor for CI, and vice versa.

Sometimes, a difference makes sense - mocking out data, or detecting hardware.

I'm asking what the speed difference is likely to be that warrants the difference between local and travis.